result_set.go

```diff
@@ -94,6 +94,11 @@ const (
 	ErrorCode_E_PARTIAL_SUCCEEDED     ErrorCode = ErrorCode(nebula.ErrorCode_E_PARTIAL_SUCCEEDED)
 )
 
+func GenResultSet(resp *graph.ExecutionResponse) (*ResultSet, error) {
+	var defaultTimezone timezoneInfo = timezoneInfo{0, []byte("UTC")}
```

Comment:
please help to check the default timezoneInfo @wenhaocs

Reply:
It's fine